src/frontend/js/components/DashboardCreditCardsManagement/index.spec.tsx

```diff
@@ -113,9 +113,7 @@ describe('<DashboardCreditCardsManagement/>', () => {
   });
 
   it('renders the correct label for an expiration date that will soon expire', async () => {
-    const limit = new Date();
-    limit.setMonth(limit.getMonth() + 1);
-    const date = faker.date.future(0.2, limit);
+    const date = faker.date.future(3 / 12); // [now < date < +3 months]
```

Comment:
```suggestion
   const offset = new Date();
    offset.setMonth(offset.getMonth() + 1);
    offset.setDate(1);
    const date = faker.date.future(2 / 12, offset);
```

To prevent flaky tests more prone during starts of months.